### PR TITLE
Show action btns in mobile, wrap links in mobile, scroll on edit action

### PR DESCRIPTION
**Pull request by mmmmillar**
**What**  

- Show action buttons in mobile
- Wrap long links in mobile
- Scroll to the edit action form when edit action clicked

### CODE REVIEW

File: components/Feature/EditAction/index.js [hunk 1]
```diff
@@ -10,6 +10,12 @@ const EditAction = ({ action, onActionUpdated, residentName, officerName }) => {
   const [description, setActionDescription] = useState(action.description);
   const [validate, setValidate] = useState(true);
 
+  const containerRef = createRef();
+
+  useEffect(() => {
+    containerRef.current.scrollIntoView({ behavior: 'smooth' });
```
Comment (t-davies) [inline components/Feature/EditAction/index.js:16]: Snazzy.

File: components/Form/Details/index.module.scss [hunk 2]
```diff
@@ -8,4 +10,8 @@
 
 .lbh-details__text {
   border-left: 0;
+
+  @media (max-width: #{$screen-sm-min}) {
+    word-break: break-word;
+  }
```
Comment (t-davies) [inline components/Form/Details/index.module.scss:16]: Not sure how long these links could get, but is it worth this on all screen resolutions do you think?

File: components/Feature/EditAction/index.js (hunk 1 above)
Comment (mmmmillar) [reply components/Feature/EditAction/index.js:16]: **behavior: 'smooth'**

something to live by

File: components/Form/Details/index.module.scss (hunk 2 above)
Comment (mmmmillar) [reply components/Form/Details/index.module.scss:16]: Hmm

Comment (mmmmillar) [reply components/Form/Details/index.module.scss:16]: I'm gonna try a supermassive link

**Pull request merged**